
---
bool connection_control
float64 distance_x
float64 distance_y
float64 distance_z